css-display-3/display-contents-dynamic-flex-001-inline.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<meta charset="utf-8">
+<title>CSS Display 3: Display contents reattachment works well in a flex container</title>
+<link rel="author" title="Emilio Cobos Álvarez" href="mailto:ecobos@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-display-3/#valdef-display-contents">
+<link rel="match" href="display-contents-flex-001-ref.html">
+<script src="./support/util.js"></script>
+<style>
+.flex {
+  display: flex;
+}
+.inline {
+  display: inline;
+}
+.contents {
+  display: contents;
+}
+</style>
+<div class="flex">
+0
+  <div class="contents">
+    <div>1</div>
```

Comment:
You should be able to see if the test passed or not by having a clear pass condition in the test case without having to look at the reference. That would make the intention of the test clearer too.